src/aura/STG_CMP_Addr/STG_CMP_Addr.cmp

```diff
@@ -63,8 +64,37 @@
         <div class="slds-col slds-size--1-of-1 slds-m-bottom--medium">
             <ui:outputText value="{!$Label.c.stgHelpContactPreferredEmail}" class="slds-text-body--small" />
         </div>
-        
+
+        <div class="slds-col slds-size--1-of-2">
+            <ui:outputText value="{!$Label.c.stgEnablePreferredPhoneSync}"/>
+        </div>
         <div class="slds-col slds-size--1-of-2">
+            <div class="slds-form-element">
+                <div class="slds-form-element__control">
+                    <label class="slds-checkbox">
+                        <aura:if isTrue="{!v.isView}">
+                            <ui:outputCheckbox value="{!v.hierarchySettings.Enable_New_Preferred_Phone_Sync__c}"
+                                               class="contact-preferred-phone-sync-enabled"
+                                               aura:id="enablePrefPhone" />
+                        <aura:set attribute="else">
+                            <ui:inputCheckbox value="{!v.hierarchySettings.Enable_New_Preferred_Phone_Sync__c}"
+                                              class="contact-preferred-phone-sync-enabled"
+                                              aura:id="enablePrefPhoneEdit" 
+                                              change="{!c.handlePhoneSync}"/>
+                            <span class="slds-checkbox--faux"></span>
+                            <span class="slds-form-element__label"></span>
+                        </aura:set>
+                        </aura:if>
+                    </label>
+                </div>
+            </div>
+        </div>
+        
+        <div class="slds-col slds-size--1-of-1 slds-m-bottom--medium">
+            <ui:outputText value="{!$Label.c.stgHelpContactPreferredPhoneSync}" class="slds-text-body--small" />
+        </div>
+        
+        <div class="slds-col slds-size--1-of-2 slds-p-left--large">
```

Comment:
FYI the slds-p-left--large here ensures that this setting is now a sub-setting of "Enable Enhanced Preferred Phone Functionality"